### PR TITLE
Add CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by 72636c**
It actually exists!

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,271 @@
...
+yarn skuba
+
+# prints version from local package.json
+yarn skuba version
+
+# builds skuba using itself
+yarn skuba build
+```
+
+If you want to test out any changes to **skuba** CLI on another local repo,
```
Comment (72636c) [inline CONTRIBUTING.md:148]:
```suggestion
If you want to test out any changes to the **skuba** CLI on another local repo,
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,271 @@
+# Contributing
+
+Hi there, thanks for checking out our repo!
+
+**skuba** is a toolkit for developing TypeScript backend applications and packages at SEEK.
+While third-party contributions are certainly welcome,
+this project is primarily driven by our internal priorities and technology strategy.
+
+SEEKers: this repo is public,
+so don't commit or post anything that isn't ready for the entire world to see.
```
Comment (etaoins) [inline CONTRIBUTING.md:10]: :+1:

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,271 @@
...
+
+```shell
+# greeter | koa-rest-api | ...
+yarn test:template greeter
+
+# clean up temporary sibling directory
+rm -fr ../tmp-greeter
+```
+
+If you want to test out any changes to the **skuba** CLI on itself,
```
Comment (tobyhei) [inline CONTRIBUTING.md:134]: ### Running locally

Comment (72636c) [reply CONTRIBUTING.md:134]: #134

**Pull request merged**